### DISCUSSION TITLE
Welcome to pkg-dev-psc-2024-04-29 Discussions!

### DISCUSSION THREAD

**Discussion by ateucher** [ateucher/pkg-dev-psc-2024-04-29#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clairehornby** [comment]
Hi everyone!

My name is Claire, I work as a biologist for DFO based in Winnipeg. Looking forward to learning about package development!
